package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -540,7 +540,7 @@ dhcp_add() {
 	[ static = "$proto" ] || return 0
 
 	# Override interface netmask with dhcp config if applicable
-	config_get netmask "$cfg" netmask "${subnet##*/}"
+	config_get netmask "$cfg" netmask
```

Comment:
this line can go.
`netmask` is no longer used anywhere.

Reply:
This is used to override the currently configured netmask with a custom one specified in the DHCP pool section:
https://github.com/openwrt/openwrt/blob/ae500e62e2938e112ae1fc6aa7389e8c7b784b13/package/network/services/dnsmasq/files/dnsmasq.init#L542
Are you sure we want to drop support for custom netmask?